modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -2485,11 +2668,20 @@ The function transforms a single-channel disparity map to a 3-channel image repr
 surface. That is, for each pixel (x,y) and the corresponding disparity d=disparity(x,y) , it
 computes:
 
-\f[\begin{array}{l} [X \; Y \; Z \; W]^T =  \texttt{Q} *[x \; y \; \texttt{disparity} (x,y) \; 1]^T  \\ \texttt{\_3dImage} (x,y) = (X/W, \; Y/W, \; Z/W) \end{array}\f]
```

Comment:
Missing from the original doc: how to get the 3D point by dividing by `W`.

Reply:
I added some parts on this mapping.